sdk/digitaltwins/Azure.ResourceManager.DigitalTwins/api/Azure.ResourceManager.DigitalTwins.netstandard2.0.cs

```diff
@@ -175,13 +175,30 @@ public partial class DigitalTwinsResource
         public string Location { get { throw null; } set { } }
         public string Name { get { throw null; } }
         public Azure.ResourceManager.DigitalTwins.Models.DigitalTwinsSkuInfo Sku { get { throw null; } set { } }
-        public System.Collections.Generic.IDictionary<string, string> Tags { get { throw null; } set { } }
+        public System.Collections.Generic.IDictionary<string, string> Tags { get { throw null; } }
         public string Type { get { throw null; } }
     }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
```

Comment:
Do we want this added?

Reply:
The attribute is autogenerated by the compiler. We tend to keep them in the listings.